src/app/core/components/top-bar/top-bar.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ContentInfo } from '../../../shared/models/content/content-info';
+import { delay } from 'rxjs/operators';
+import { CurrentContentService } from '../../../shared/services/current-content.service';
+import { Mode, ModeAction, ModeService } from '../../../shared/services/mode.service';
+
+@Component({
+  selector: 'alg-top-bar',
+  templateUrl: './top-bar.component.html',
+  styleUrls: [ './top-bar.component.scss' ],
+})
+export class TopBarComponent {
+  @Input() scrolled = false;
+  @Input() currentMode?: Mode;
+  @Input() fullFrameContent?: boolean;
```

Comment:
boolean inputs should have `false` as default value, like `scrolled` 2 lines upper

Reply:
Fixed

Reply:
It doesn't look fixed, do you have unpushed commits or something ? :thinking: 

By the way, it also applies for [`ContentTopBarComponent`](https://github.com/France-ioi/AlgoreaFrontend/pull/686/files#diff-81c946e33acda69df58892a90c0e515df6b361f78f5cffd3ea92a83aef13bdcd)